barretenberg/cpp/src/barretenberg/op_queue/ecc_op_queue.hpp

```diff
@@ -193,14 +193,14 @@ class ECCOpQueue {
     }
 
     /**
-     * @brief Write no op (i.e. empty row)
+     * @brief Writes a no op (i.e. two zero rows) to the ultra ops table but no eccvm operations.
      *
+     * @return We want to be able to add  data (eventually, randomness) to the ultra ops table to hide commitments sent
+     * to the rollup as part of the merge protocol without affecting the operations in the ECCVM.
      */
```

Comment:
not entirely sue why we hav a no-op operation, it's not something we need and was only use for testing in ECCVM (prolly added for development purposes) - removed to avoid confusion.